src/EventStore.Core/Configuration/ClusterVNodeOptions.cs

```diff
@@ -364,6 +364,8 @@ public record DatabaseOptions {
 			public bool Unbuffered { get; init; } = false;
 
 			[Description("The initial number of readers to start when opening a TFChunk.")]
+			[Deprecated("This configuration property has no effect")]
+			[Obsolete("This configuration property has no effect")]
```

Comment:
```suggestion
			[Description("The initial number of readers to start when opening a TFChunk.")]
			[Deprecated("The ChunkInitialReaderCount parameter has been deprecated as of version 24.6.0 and currently has no effect.")]
```
Just for proper logging and consistency with other deprecated options.
Also, it appears that the `Obsolete` attribute is set only when a parameter has been renamed, so we may not need it here.